src/plotman/archive.py

```diff
@@ -49,8 +49,14 @@ def compute_priority(phase, gb_free, n_plots):
 
 def get_archdir_freebytes(arch_cfg):
     archdir_freebytes = { }
-    df_cmd = ('ssh %s@%s df -aBK | grep " %s/"' %
-        (arch_cfg['rsyncd_user'], arch_cfg['rsyncd_host'], arch_cfg['rsyncd_path']) )
+    arch_mode = arch_cfg.get('mode', 'remote')
```

Comment:
If this is consistent with other defaults, ok.  Though it really should be handled centrally so we don't accidentally default differently in different places.  But, any real fix of this system as a whole should be handled separately and is probably related to #77.

Reply:
No matter which comes first but there's https://github.com/ericaltendorf/plotman/pull/98 and wherever that comes together with this we can make get this default set at a single point.